qiskit/providers/ibmq/ibmqfactory.py

```diff
@@ -28,13 +27,12 @@
 from .credentials.updater import update_credentials
 from .exceptions import IBMQAccountError, IBMQApiUrlError, IBMQProviderError
 
-
 logger = logging.getLogger(__name__)
 
 QX_AUTH_URL = 'https://auth.quantum-computing.ibm.com/api'
 UPDATE_ACCOUNT_TEXT = (
-    'Please update your accounts and programs by following the instructions here:\n'
-    'https://github.com/Qiskit/qiskit-ibmq-provider#updating-to-the-new-ibm-q-experience')
+    "Please update your accounts and programs by following the instructions here:",
+    "https://github.com/Qiskit/qiskit-ibmq-provider#updating-to-the-new-ibm-q-experience")
```

Comment:
Small note - with the changes mentioned on the main comment, probably a ` ` needs to be introduced before the url.